Changes

```diff
@@ -1,5 +1,17 @@
 Release history for Zonemaster component Zonemaster-GUI
 
+
+v4.4.0 2025-03-04 (part of Zonemaster v2024.2.1 release)
```

Comment:
Maybe "introduced in" would be a better wording than "part of"? I'm thinking sometimes a component version is reused for multiple product releases.

Reply:
It is the same wording as last release. Let us look at the wording for next release. I think that "part of" is better than "introduced in", but maybe we could find a another wording that works better.